spark-3.1-spanner-lib/src/test/java/com/google/cloud/spark/spanner/acceptance/DataprocAcceptanceTestBase.java

```diff
@@ -0,0 +1,356 @@
+package com.google.cloud.spark.spanner.acceptance;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.dataproc.v1.*;
+import com.google.cloud.spanner.Database;
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Instance;
+import com.google.cloud.spanner.InstanceAdminClient;
+import com.google.cloud.spanner.InstanceConfig;
+import com.google.cloud.spanner.InstanceId;
+import com.google.cloud.spanner.InstanceInfo;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+import com.google.cloud.spark.spanner.TestData;
+import com.google.common.base.Preconditions;
+import com.google.spanner.admin.database.v1.CreateDatabaseMetadata;
+import com.google.spanner.admin.instance.v1.CreateInstanceMetadata;
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import org.junit.Test;
+
+/** */
+public class DataprocAcceptanceTestBase {
+
+  // public static final String CONNECTOR_JAR_DIRECTORY = "target";
+  public static final String CONNECTOR_JAR_DIRECTORY =
+      "/usr/local/google/home/halio/Program/brand_new/spark-spanner-connector/spark-3.1-spanner/target";
```

Comment:
We can use relative file paths by:
* getting the directory that this code is running in: it'll be the root directory that we run ./mvnw from then adding the suffix spark-3.1-spanner-lib/target

Or if programmatically you can get by means of reflection the runtime caller file of this method at a program counter offset of 0, it'll be file and you can go backwards with ".."